llvm/test/CodeGen/X86/avx512-cmp.ll

```diff
@@ -155,10 +155,10 @@ B:
 define i32 @test10(i64 %b, i64 %c, i1 %d) {
 ; ALL-LABEL: test10:
 ; ALL:       ## %bb.0:
+; ALL-NEXT:    xorb $1, %dl
```

Comment:
Is this a code size regression? I can't remember if `notb` is shorter than `xorb`

Reply:
It is one byte bigger. I guess I can just say no for i1

Reply:
This was not triggering before because this only ran post-legalization.

Reply:
I think this is just a problem of the backend not choosing not over xor 1 when none of the bits are demanded anyway: